components/combobox/stories/template.js

```diff
@@ -26,15 +25,9 @@ const Combobox = ({
 	isFocused = false,
 	isKeyboardFocused = false,
 	isLoading = false,
-	selectedDay,
+	value,
```

Comment:
Does this need to be set to an empty string, like it is in `Template`? `value = ""`? Or is that `value = ""` being an empty string specific to a combobox with a popover? I'm a little fuzzy on how to use `Combobox` vs `Template` 😬

Reply:
Looks like we're not exporting `Combobox` so the only place it's used is within the template, which will get a default value if none is set. I probably didn't set it because of that, thinking it'd be fine without it, but I don't think it hurts to add it, and if some part of the code gets changed so that the value isn't being passed to `Combobox`, we're covered.